elements/pl-order-blocks/pl-order-blocks.py

```diff
@@ -345,22 +366,45 @@ def render(element_html, data):
         check_indentation = pl.get_boolean_attrib(element, 'indentation', INDENTION_DEFAULT)
         indentation_message = ', with correct indentation' if check_indentation is True else None
 
-        question_solution = [{
-            'inner_html': solution['inner_html'],
-            'indent': ((solution['indent'] or 0) * TAB_SIZE_PX) + INDENT_OFFSET
-        } for solution in data['correct_answers'][answer_name]]
-
-        html_params = {
-            'true_answer': True,
-            'question_solution': question_solution,
-            'ordering_message': ordering_message,
-            'indentation_message': indentation_message,
-            'block_formatting': block_formatting
-        }
-        with open('pl-order-blocks.mustache', 'r', encoding='utf-8') as f:
-            html = chevron.render(f, html_params)
-        return html
+        distractor_info = pl.get_string_attrib(element, 'distractor-info', DISTRACTOR_INFO_DEFAULT)
+        if distractor_info != 'none':
+            all_distractors = [item for item in data['params'][answer_name] if not item['is_correct']]
+            for distractor in all_distractors:
+                if (distractor['distractor-feedback'] is None) and (distractor_info == 'show-feedback'):
+                    raise Exception('Feedback must be associated with each distractor to use the show-feedback option')
+
+                if distractor_info == 'show-distractors' or distractor_info == 'none':
+                    distractor['margin'] = '0px'
+                elif distractor_info == 'show-feedback':
+                    distractor['margin'] = '10px'
+                else:
+                    raise Exception('The distractor info tag must be show-distractors, show-feedback or none')
 
+        else:
+            all_distractors = []
+
+        if answer_name in data['correct_answers']:
```

Comment:
do we need this if check here? Even if there is a possibility that the key is not in the dictionary, doing a call to `get` with a default value is a nicer solution than having this whole `if` statement.

Reply:
Sorry, for this one, are you referring to the if on line 386 or the if-else blocks above it?

Reply:
Ya, line 386.